### PR TITLE
set the dart_appwrite package version to 8.0.0

### LINKED ISSUE #5521: 🐛 Bug Report: Appwrite cli initializes dart function with old dart_appwrite version
### 👟 Reproduction steps

When I create a dart function using the cli, the dart_appwrite version is set to 7.1.0 instead of 8.0.0

### 👍 Expected behavior

It should set the dart_appwrite package version to 8.0.0

### 👎 Actual Behavior

It sets the dart_appwrite package version to 7.1.0

### 🎲 Appwrite version

Version 1.3.x

### 💻 Operating system

Linux

### 🧱 Your Environment

_No response_

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/.github/blob/main/CODE_OF_CONDUCT.md)

### PR DESCRIPTION
**Pull request by PoorviShetty**
## What does this PR do?

Set the dart_appwrite package version to 8.0.0

## Related PRs and Issues

* https://github.com/appwrite/appwrite/issues/5521

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

**Review by stnguyen90 — APPROVED** [review]
I think this is good because main is used for Appwrite up to version 1.3 and version 8.0.0 is meant for Appwrite 1.3.

File: dart-2.16/pubspec.yaml [hunk 1]
```diff
@@ -6,8 +6,8 @@ publish_to: 'none'
 environment:
   sdk: '>=2.16.0 <3.0.0'
 
-dependencies: 
-  dart_appwrite: ^7.1.0
-      
+dependencies:
+  dart_appwrite: ^8.0.0
+
```
Comment (stnguyen90) [inline dart-2.16/pubspec.yaml]: It looks like the dart_appwrite package [version 7 and above require dart 2.17](https://pub.dev/packages/dart_appwrite/versions), so let's downgrade the versions for everything under dart 2.17 to ^6.0.1

**Review by stnguyen90 — CHANGES_REQUESTED** [review]
Great PR! 🤯 We left some comments during the review, please check them out.

File: dart-2.16/pubspec.yaml (hunk 1 above)
Comment (PoorviShetty) [reply dart-2.16/pubspec.yaml]: Hi, I've pushed a commit with the required changes

**Pull request closed without merging**